config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits-release-1.7.yaml

```diff
@@ -22,7 +22,7 @@ presubmits:
     annotations:
       testgrid-dashboards: vmware-cluster-api-provider-vsphere, sig-cluster-lifecycle-cluster-api-provider-vsphere
       testgrid-tab-name: pr-test-release-1-7
-      testgrid-alert-email: k8s-testing-clusterapi-vsphere+alerts@groups.vmware.com
+      testgrid-alert-email: sig-cluster-lifecycle-cluster-api-vsphere-alerts@kubernetes.io
```

Comment:
We also need testgrid-num-failures-to-alert here

Reply:
As discussed: not on presubmits :-)